src/AppBundle/Controller/AuthenticationController.php

```diff
@@ -80,7 +80,7 @@ public function authenticationAction(Request $request)
         if (!$this->authenticationService->authenticationRequired()) {
             $logger->error('There is no pending authentication request from SP');
 
-            throw new NoActiveAuthenrequestException('Registration is not required');
+            throw new NoActiveAuthenrequestException();
```

Comment:
Consider moving the error log message (2 lines above) into the exception and remove the log entry. All exceptions should be logged right? No need to do this twice.